spectrum-network/src/peer_manager/data.rs

```diff
@@ -42,7 +42,17 @@ impl Into<DialOpts> for PeerDestination {
 
 #[derive(Debug, Clone, Copy, PartialEq, Eq)]
 pub enum ReputationChange {
-    MalformedMessage = 10,
```

Comment:
Have enum variants as integer type means that each variant must take a unique value. Might not want that for our use case.